### PR TITLE
List members

### PR DESCRIPTION
**Pull request by linusarnejohansson**
- components to list active invites and members on a organization.
- backwards compatible

- code for edit user commented out because it is not backwards compatible with mode 0.5.x update
- code for removing user commented out because it is not backwards compatible with mode 0.5.x update

- edit invite hidden because it is not backwards compatible with mode 0.5.x update
- remove invite hidden because it is not backwards compatible with mode 0.5.x update

### CODE REVIEW

File: src/components/user/list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@Component({
+	tag: "userwidgets-user-list",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class UserwidgetsUserList {
+	@Prop() state: model.State
+	@State() users?: userwidgets.User.Readable[]
+	@State() key?: userwidgets.User.Key
+	@Prop({ mutable: true }) organization?: userwidgets.Organization
```
Comment (EliasEriksson) [inline src/components/user/list/index.tsx:16]: does this need to be mutable?

File: src/components/user/list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	@State() translate: langly.Translate = translation.create("en")
+	componentWillLoad() {
+		this.state.users.listen("value", users => (this.users = users || undefined))
+		this.state.me.listen("key", async key => (this.key = key || undefined))
+		this.state.locales.listen("language", language => (this.translate = translation.create(language)))
+	}
+
+	render() {
+		const users = this.organization
```
Comment (EliasEriksson) [inline src/components/user/list/index.tsx:25]: store this result into the users state variable. add the filter function into the subscription of users.

File: src/components/user/list/invited/index.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export class UserwidgetsUserListInvited {
 	@State() key?: userwidgets.User.Key
 	@State() organization?: userwidgets.Organization
 	@State() organizations?: userwidgets.Organization[]
-	@State() users?: userwidgets.User.Readable[]
+	@State() users?: userwidgets.User.Readable[] // do we really get this?
```
Comment (EliasEriksson) [inline src/components/user/list/invited/index.tsx:16]: what does this comment mean?

File: src/components/user/list/invited/index.tsx [hunk 4]
```diff
@@ -32,6 +32,18 @@ export class UserwidgetsUserListInvited {
...
 	}
+	reInvite(user: string) {
+		const users = this.organization?.users.map(e => (e == user ? { user: user } : e))
+		if (this.organization)
+			this.state.organizations.update(this.organization.id, { users: users })
+	}
+	removeInvitation(user: string) {
+		const users = this.organization?.users.filter(e => e != user)
+		if (this.organization) {
+			this.state.organizations.update(this.organization.id, { users: users })
```
Comment (EliasEriksson) [inline src/components/user/list/invited/index.tsx:43]: tell the users if this is successful or fails.

File: src/components/user/list/invited/index.tsx [hunk 5]
```diff
@@ -32,6 +32,18 @@ export class UserwidgetsUserListInvited {
 						.find(organization => organization.id == this.organization?.id)
 						?.users.filter(email => email != this.key?.email && !this.users?.find(user => user.email == email)) ?? []
 	}
+	reInvite(user: string) {
+		const users = this.organization?.users.map(e => (e == user ? { user: user } : e))
+		if (this.organization)
+			this.state.organizations.update(this.organization.id, { users: users })
```
Comment (EliasEriksson) [inline src/components/user/list/invited/index.tsx:38]: tell the user if this is successful or fails.

File: src/components/user/list/invited/style.css [hunk 6]
```diff
@@ -7,7 +7,9 @@
 }
 
 .inputs {
-	display: flex;
+	height: 0px;
+	width: 0px;
+	/* display: flex; */
```
Comment (EliasEriksson) [inline src/components/user/list/invited/style.css:12]: remove unused

**Review by EliasEriksson — CHANGES_REQUESTED** [review]

File: src/components/user/list/invited/index.tsx (hunk 5 above)
Comment (linusarnejohansson) [reply src/components/user/list/invited/index.tsx:38]: the feature is deactivated and not fully implemented. Waiting for model 0.5.x until the functionality can be fully implemented.

File: src/components/user/list/invited/index.tsx (hunk 4 above)
Comment (linusarnejohansson) [reply src/components/user/list/invited/index.tsx:43]: the feature is deactivated and not fully implemented. Waiting for model 0.5.x until the functionality can be fully implemented.

**Review by EliasEriksson — APPROVED** [review]

**Pull request merged**